packages/code-du-travail-frontend/src/search/SearchResults/Results.js

```diff
@@ -98,7 +98,9 @@ export const Results = ({ id, isSearch, items, query }) => {
       aria-label="Résultats de recherche"
     >
       {isSearch ? (
-        <Heading id={id}>{`Résultats de recherche pour “${query}”`}</Heading>
```

Comment:
Les marges de Heading ont étés revues à la hausse dans un PR précédente, je crois que ce changement n'a plus lieu d'être ^^

Reply:
OK c'était noté dans la liste à faire je vais rechanger